skyvern/client/client.py

```
@@ -1577,6 +1593,14 @@ def workflows(self):
             self._workflows = WorkflowsClient(client_wrapper=self._client_wrapper)
         return self._workflows
 
+    @property
+    def browser_profiles(self):
+        if self._browser_profiles is None:
+            from .browser_profiles.client import BrowserProfilesClient  # noqa: E402
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)